### PR TITLE
explain the simple stof solution first

### PR DESCRIPTION
**Pull request by OskarStark**
in my case, i injected the service and at the next step i realized that this was not necessary in my case, because i had enabled the stof doctrine extensions bundle.

### CODE REVIEW

File: Resources/doc/cookbook/recipe_sortable_listing.rst [hunk 1]
```diff
@@ -38,8 +38,17 @@ First of all we are going to add a position field in our ``Client`` entity.
...
-Then we need to inject the Sortable listener. If you only have the Gedmo bundle enabled,
-add the listener to your config.yml
+If you have the ``stof/doctrine-extensions-bundle``, you only need to enable the sortable
+feature in your config.yml such as
+
+.. code-block:: yaml
+
+	stof_doctrine_extensions:
+	    orm:
+                default:
```
Comment (rande) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: something is wrong is the alignement

Comment (OskarStark) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: indeed, i just copy paste this block, will fix the indention

**Comment by OskarStark** [comment]
oops, something went wrong

**Comment by core23** [comment]
Please rebase @OskarStark

**Comment by OskarStark** [comment]
i already reworded it

**Pull request closed without merging**